test/test_ca.py

```diff
@@ -15,6 +18,65 @@ def local_ca_fixture(tmp_path_factory, backup_sssd_ca_db):
     return ca
 
 
+@pytest.fixture()
+def dummy_ipa_vals(ipa_ip, ipa_hostname, ipa_admin_passwd, ipa_root_passwd):
+    """
+    Creates dummy values for IPA serve and client for testings
+    """
+    domain = ipa_hostname.split(".", 1)[1]
+    return {
+        "server_ip": ipa_ip,
+        "server_domain": domain,
+        "server_hostname": ipa_hostname,
+        "server_admin_passwd": ipa_admin_passwd,
+        "server_realm": domain.upper(),
+        "server_root_passwd": ipa_root_passwd,
+        "client_hostname": f"client-hostname.{domain}"
+    }
+
+
+@pytest.fixture()
+def dummy_user():
+    class User:
+        def __init__(self):
+            self.username = f"username-{randint(1, 100)}"
+            self.password = f"password-{self.username}"
+
+    return User()
+
+
+@pytest.fixture()
+def ipa_meta_client(dummy_ipa_vals):
+    """
+    Return ready-to-use IPA MetaClient with admin login. This fixture might not
+    work if there is no mapping rule on your system for given IPA IP address and
+    IPA hostnames (no corresponding entry in /etc/hosts)
+    """
+    client = ClientMeta(dummy_ipa_vals["server_hostname"], verify_ssl=False)
+    client.login("admin", dummy_ipa_vals["server_admin_passwd"])
+    return client
+
+
+@pytest.fixture()
+def installed_ipa(dummy_ipa_vals, clean_ipa):
+    check_output(["ipa-client-install", "-p", "admin",
```

Comment:
you're not doing anything with the output

Reply:
yes. because there is nothing important for tests. what is your point here?

Reply:
use `subprocess.run(... check=True)`, maybe?

Reply:
`check_output` is actually a kind of wrapper over `run` function which has some parameters already set. Check documentation [here](https://docs.python.org/3/library/subprocess.html#subprocess.check_output)